.github/workflows/windows.yml

```diff
@@ -0,0 +1,81 @@
+name:                 continuous-intergration/windows
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+    paths-ignore:
+      - 'README.md'
+
+jobs:
+  check:
+    name:              build-contract-template
+    strategy:
+      matrix:
+        platform:
+          - windows-latest
+        toolchain:
+          - nightly
+    runs-on:           ${{ matrix.platform }}
+    env:
+      RUST_BACKTRACE:  full
+    steps:
+      - name:          Install sudo for windows #https://github.com/actions/virtual-environments/issues/572
+        run: choco install sudo
+
+      - name:          Set cache_hash ENV and prepare cache dir
+        run:           |
+          echo "cache_hash=${{ runner.os }}-${{ steps.toolchain.outputs.rustc_hash }}-${{ matrix.compiler }}-${{ hashFiles('**/Cargo.toml') }}" >> "$GITHUB_ENV"
+          sudo chmod -R a+w $HOME/.cargo
+        shell:         bash
+
+      - name: Cache cargo registry
```

Comment:
Indentations are not matching with some other steps which are "aligned"

Reply:
Argh, I just applied Intellij's auto-formatting and pushed the result.